packages/backend-dynamic-feature-service/src/scanner/plugin-scanner.ts

```diff
@@ -136,6 +136,11 @@ export class PluginScanner {
       withFileTypes: true,
     })) {
       const pluginDir = dirEnt;
+
+      if (pluginDir.name === 'lost+found') {
+        this.logger.info(`skipping '${pluginDir.name}' system directory`);
```

Comment:
Arguably this is unnecessary noise. _Possibly_ at debug level?

Reply:
I agree that for this case `debug` would be sufficient.
Skipping it would even be OK to me, because having a plugin package sub-folder be named `lost+found` on purpose is totally improbable.